client-react/src/authentication/index.js

```diff
@@ -65,6 +68,23 @@ const Button = styled(DynamicButton)`
   border-radius: 37px;
   border: 1px solid transparent;
   cursor: pointer;
+  color: #fff;
+  background-image: ${(props) =>
+    props.light
+      ? "-webkit-linear-gradient(90deg, #a5b8b5 50%, transparent 50%)"
+      : "-webkit-linear-gradient(90deg, #0b332f 50%, transparent 50%)"};
+  background-image: ${(props) =>
+    props.light
+      ? "linear-gradient(90deg, #a5b8b5 50%, transparent 50%)"
+      : "linear-gradient(90deg, #0b332f 50%, transparent 50%)"};
+  background-position: 100%;
+  background-size: 400%;
+  -webkit-transition: background 300ms ease-in-out;
+  transition: background 300ms ease-in-out;
+
+  &:hover {
+    background-position: 0;
+  }
```

Comment:
![image](https://user-images.githubusercontent.com/15658088/112919769-9308a200-90bc-11eb-9fbe-91ea70348848.png)

I like what you did with the animation, but the sign up text is really hard to read. Can you revert the color back to green like how it is in prod? It's also hard to read sign in when you are on the Sign Up side.